taskcluster/ci/raptor/kind.yml

```diff
@@ -20,6 +21,18 @@ job-defaults:
             armeabi-v7a: t-bitbar-gw-perf-g5
             arm64-v8a: t-bitbar-gw-perf-p2
     description: ""
+    notify:
+        by-level:
+            '3':
+                email:
+                    content: This calls for an action of the Performance team. Use the link to view it on Treeherder.
```

Comment:
Perhaps this phrasing can be improved to "This calls for action by the Performance team"?

Reply:
👍 done!